docs/sphinx/install.rst

```diff
@@ -141,6 +141,34 @@ we refer to the `CUDA Quantum GitHub repository`_.
 
 .. _CUDA Quantum GitHub repository: https://github.com/NVIDIA/cuda-quantum/blob/main/Building.md
 
+
+CUDA Quantum Dependencies
+-------------------------
+
+CUDA Quantum can be used to simulate quantum programs (see `using/simulators`) on a CPU-only system, but a GPU is highly recommended.
+
+The supported CPUs include both the x86_64 (x86-64-v3 when using the OpenMPI stack) and ARM64 architectures.
+
+For simulation of CUDA Quantum programs on NVIDIA GPUs, additional packages should be installed. See also `cuQuantum <https://docs.nvidia.com/cuda/cuquantum/latest/getting_started.html>`__.
+
+The following table summarizes the required components.
+
+.. list-table:: Requirements
```

Comment:
What do you think about breaking this into 2 tables? One to hold the true requirements (CPU and Operating System), and the other to hold the "Requirements for GPU-Accelerated Simulations"? I would hesitate to call the GPU ones _requirements_ because the statement above says "CUDA Quantum can be used to simulate quantum programs on a CPU-only system".

Reply:
How about something like "supported versions/models/thingies"?